From 2a7d7350e495026ea922d3f2e94620d9572edd3b Mon Sep 17 00:00:00 2001
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Mon, 19 Mar 2007 19:15:22 -0400
Subject: [PATCH] [PATCH] Unionfs: sioq not __exit

stop_sioq() is called from both __init and __exit functions, so it
shouldn't be marked __exit.

Reported on the kernelnewbies mailing list, but no patch offered there.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Josef 'Jeff' Sipek <jsipek@cs.sunysb.edu>
---
 fs/unionfs/sioq.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fs/unionfs/sioq.c b/fs/unionfs/sioq.c
index 7830b89..c0d89a3 100644
--- a/fs/unionfs/sioq.c
+++ b/fs/unionfs/sioq.c
@@ -40,7 +40,7 @@ int __init init_sioq(void)
 	return err;
 }
 
-void __exit stop_sioq(void)
+void stop_sioq(void)
 {
 	if (superio_workqueue)
 		destroy_workqueue(superio_workqueue);
-- 
1.5.4.3

